WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -438,10 +448,12 @@ private extension PointOfSaleAggregateModel {
 extension PointOfSaleAggregateModel {
     @MainActor
     func checkOut() async {
+        collectOrderPaymentAnalyticsTracker.trackCheckoutTapped()
```

Comment:
It doesn't get reset when we close POS. Maybe it could be considered intentional but probably not. My understanding is that all the events should reset when customer interaction starts. That would be the clearest indication from which all the other events could derive. However, it needs to be synced with android.

It could reset:
- At the same time when customer interaction starts (item is added to empty cart)
- When payment finishes
- When POS launches

Reply:
> It doesn't get reset when we close POS

I don't think it should, but reset only after the payment finishes. I'll check this one with Android as well just to be sure we're tracking the same thing. The whole "user interaction" or "session" definition is not very solid at the moment.

Reply:
Ref: p1739867406250859?thread_ts=1739867347.923049&cid=C070SJRA8DP-slack-C070SJRA8DP

Reply:
> I don't think it should, but reset only after the payment finishes.

That's why we need to sync. For me, it makes sense to only track it in the scope of the session, which would make it reset if we detect new first user interaction. But I understand that others may think differently.

Reply:
Thanks! I've updated this on https://github.com/woocommerce/woocommerce-ios/pull/15148/commits/efff5ea79fa41bb0cc301b3fa31b18f4dc6a8f9e , so we reset all ongoing counters when anything considered a "user interaction starts" happens.